flink-connectors/flink-hadoop-compatibility/src/main/java/org/apache/flink/api/java/hadoop/mapred/wrapper/HadoopInputSplit.java

```diff
@@ -70,8 +77,8 @@ public HadoopInputSplit(int splitNumber, org.apache.hadoop.mapred.InputSplit hIn
 	public String[] getHostnames() {
 		try {
 			return this.hadoopInputSplit.getLocations();
-		}
-		catch (IOException e) {
+		} catch (IOException e) {
+			e.printStackTrace();
```

Comment:
will this cause lots of messages?

Reply:
Sorry, it is just for debug. I was thinking about why to return 'new string [0]' here. Later, I found that this is a convention of `InputSplit`. They all do this. Maybe they think this is just an optimization.